pyrk/materials/material.py

```diff
@@ -10,15 +10,18 @@ class Material(object):
     def __init__(self, name=None,
                  k=0*units.watt/units.meter/units.kelvin,
                  cp=0*units.joule/units.kg/units.kelvin,
+                 mu=0*units.pascal*units.seconds,
```

Comment:
should current material classes (such as flibe and graphite) be updated to include mu?